hail/src/test/scala/is/hail/services/shuffler/ShuffleSuite.scala

```diff
@@ -26,240 +26,208 @@ class ShuffleSuite extends HailSuite {
 
-          val left = struct(0)
-          val right = struct(nElements)
+        val left = struct(0)
+        val right = struct(nElements)
 
-          val fromOneQuery = arrayOfUnsafeRow(
-            rowDecodedPType,
-            c.get(region, left, true, right, false))
+        val fromOneQuery = arrayOfUnsafeRow(
+          rowDecodedPType,
+          c.get(region, left, true, right, false))
 
-          i = 0
-          val ab = new ArrayBuilder[Long]()
-          while (i < nPartitions) {
-            ab ++= c.get(region,
-              partitionBounds(i).offset, true,
-              partitionBounds(i + 1).offset, i == nPartitions - 1)
-            i += 1
-          }
-          val fromPartitionedQueries = arrayOfUnsafeRow(
-            rowDecodedPType,
-            ab.result())
+        i = 0
+        val ab = new ArrayBuilder[Long]()
+        while (i < nPartitions) {
+          ab ++= c.get(region,
+            partitionBounds(i).offset, true,
+            partitionBounds(i + 1).offset, i == nPartitions - 1)
+          i += 1
+        }
+        val fromPartitionedQueries = arrayOfUnsafeRow(
+          rowDecodedPType,
+          ab.result())
 
-          i = 0
-          def assertFirstFieldEqualsIndex(rows: IndexedSeq[UnsafeRow], i: Int): Unit = {
-            assert(!rows(i).isNullAt(0),
-              s"""first field is undefined ${rows(i)}
+        i = 0
+        def assertFirstFieldEqualsIndex(rows: IndexedSeq[UnsafeRow], i: Int): Unit = {
+          assert(!rows(i).isNullAt(0),
+            s"""first field is undefined ${rows(i)}
                |Context: ${rows.slice(i-3, i+3)}.
                |Length: ${rows.length}""".stripMargin)
-            assert(rows(i).getInt(0) == i,
-              s"""first field should be ${i}: ${rows(i)}.
+          assert(rows(i).getInt(0) == i,
+            s"""first field should be ${i}: ${rows(i)}.
                |Context: ${rows.slice(i-3, i+3)}.
                |Length: ${rows.length}""".stripMargin)
-          }
-          while (i < nElements) {
-            assertFirstFieldEqualsIndex(fromOneQuery, i)
-            assertFirstFieldEqualsIndex(fromPartitionedQueries, i)
-            i += 1
-          }
-          assert(fromOneQuery.length == nElements)
-          assert(fromPartitionedQueries.length == nElements)
         }
-        c.stop()
-      }
-    } finally {
-      if (server != null) {
-        server.stop()
+        while (i < nElements) {
+          assertFirstFieldEqualsIndex(fromOneQuery, i)
+          assertFirstFieldEqualsIndex(fromPartitionedQueries, i)
+          i += 1
+        }
+        assert(fromOneQuery.length == nElements)
+        assert(fromPartitionedQueries.length == nElements)
       }
+      c.stop()
     }
   }
 
   @Test def testShuffleIR() {
-    var server = new ShuffleServer(sslContext(
-      "src/test/resources/non-secret-key-and-trust-stores/server-keystore.p12",
-      "hailhail",
-      "PKCS12",
```

Comment:
Did you mean to not actually test this?

Reply:
Yes, this PR tests the shuffler service itself rather than starting a server in the test container to test against.